ember-cli-build.js

```diff
@@ -1,13 +1,15 @@
-/* global require, module */
+/* eslint-env node */
+'use strict';
+
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
-  const app = new EmberAddon(defaults, {
+  let app = new EmberAddon(defaults, {
```

Comment:
Need to change this back to `const` to make CodeClimate happy.

Reply:
got it! I'll update.